scripts/nbrshow

```diff
@@ -111,7 +111,10 @@ class NbrBase(object):
         rc = p.wait()
 
         if rc == 0:
-            rawdata = output
+            if isinstance(output, bytes):
+                rawdata = output.decode()
+            else:
+                rawdata = output
```

Comment:
Thanks for catching this. I must have missed it. However, rather than this change, please add the parameter `text=True` to the `Popen` call on line 109 instead. E.g.,

```python3
p = subprocess.Popen(self.cmd, shell=True, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

This will force the output to be strings, not bytes.


p = subprocess.Popen(self.cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)

Reply:
I considered this method too. But the ```text=True``` is only available on python 3. I'm afraid we might run this script on python 2 and the call will trigger another exception. So I select current way.

Reply:
sonic-utilities has been converted to Python 3 in the master branch. We are no longer supporting Python 2 moving forward. You can make this change instead.

Reply:
Sure. Updated. Thanks